### PR TITLE
Refactor console special characters

### PR DESCRIPTION
**Pull request by lokson**
Gather definitions of colors and special characters in log.js.

Closes: https://github.com/GoogleChrome/lighthouse/issues/1414

### CODE REVIEW

File: lighthouse-core/lib/log.js [hunk 1]
```diff
@@ -129,7 +129,40 @@ class Log {
...
+   * @param {string} str
+   * @return {string}
+   */
+  static redify(str) {
+    return `${Log.red}${str}${Log.reset}`;
+  }
+
 }
+
+Log.green = '\x1B[32m';
```
Comment (wardpeet) [inline lighthouse-core/lib/log.js]: Any reason why you don't make this constants? They don't need to be on the class.

Comment (lokson) [inline lighthouse-core/lib/log.js]: Having them in class doesn't require excessive import statements like `import {green, red, bold ... `. I don't mind changing this if you prefer named exports for each of those.

Comment (wardpeet) [inline lighthouse-core/lib/log.js]: no not exports :)

but not my call to make so waiting for the rest of the team to give final feedback

```
const red = '\x1B[31m';
const reset = '\x1B[0m';

class Log {
...
static redify(str) {
  return `${red}${str}${reset}`;
}
module.exports = Log;
```

Comment (XhmikosR) [inline lighthouse-core/lib/log.js]: 👍 for @wardpeet's approach.

Comment (lokson) [inline lighthouse-core/lib/log.js]: I agree that would be the best, but definitions (of tick, yellow and other) are used outside this module for example in print.ts so I guess it has to go into export somehow.

Comment (lokson) [inline lighthouse-core/lib/log.js]: btw. that was the whole point of this issue: to cut duplications.

**Comment by XhmikosR** [comment]
@lokson: thanks for this, I had started doing it locally so this saved me some time :)

File: lighthouse-core/lib/log.js (hunk 1 above)
Comment (XhmikosR) [inline lighthouse-core/lib/log.js]: Hmm, indeed that's a fair point.

So, it's the decision of either class or exports.

**Comment by lokson** [comment]
@XhmikosR: welcome!

File: lighthouse-core/lib/log.js (hunk 1 above)
Comment (ebidel) [inline lighthouse-core/lib/log.js]: I like the idea of using language constructs (static getters).

Comment (lokson) [inline lighthouse-core/lib/log.js]: @ebidel, right something like:

```js
class Log {
   static get green() { return '\x1B[32m'; }
}
```

I'll send a commit.

**Comment by paulirish** [comment]
👩‍🎨🎨

**Pull request merged**